packages/vite/package.json

```diff
@@ -1,18 +1,29 @@
 {
   "name": "vite",
   "version": "3.0.0-alpha.1",
+  "type": "module",
   "license": "MIT",
   "author": "Evan You",
   "description": "Native-ESM powered web dev build tool",
   "bin": {
     "vite": "bin/vite.js"
   },
-  "main": "dist/node/index.js",
-  "types": "dist/node/index.d.ts",
+  "main": "./dist/node/index.js",
+  "module": "./dist/node/index.js",
+  "types": "./dist/node/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/node/index.d.ts",
+      "import": "./dist/node/index.js",
+      "require": "./index.cjs"
+    },
+    "./*": "./*"
```

Comment:
Wonder if we can be more strict and remove this. We can then add `./client` for the typescript types `client.d.ts`?

Reply:
Agree, we could remove it here and open as needed.